backend/src/main/java/com/gildedrose/ItemsController.java

```diff
@@ -43,6 +43,16 @@ public ItemsController(Datasource<Ageable> hashMapDB) {
                 .build();
     };
 
+    final Function<Request, Response> okAllItemsUpdateHandler = request -> {
```

Comment:
Question: Do we have a test for this controller?

Reply:
I do now! [eea17a9](https://github.com/jramnani/GildedRoseJavaLLC/pull/34/commits/eea17a9519f2eaf8b6296a4e8b5702df7fd08603)